xml/templates/message_templates/event_offline_receipt_html.tpl

```diff
@@ -37,7 +37,7 @@
       <p>{ts}Once your registration has been reviewed, you will receive an email with a link to a web page where you can complete the registration process.{/ts}</p>
      {/if}
     {elseif !empty($is_pay_later)}
```

Comment:
@eileenmcnaughton is there a reason why your not getting rid of this empty as per L304?

Reply:
@seamuslee001 I didn't spot it - if you want to merge this I'll include it in the next one due to the painfulness of the whole sql regen thing

Reply:
@eileenmcnaughton ok and there is a corresponding one in the txt at L28